src/test/java/ru/training/at/hw2/ex1/Exercise1.java

```diff
@@ -0,0 +1,99 @@
+package ru.training.at.hw2.ex1;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+public class Exercise1 {
+    private WebDriver webDriver;
+
+    @BeforeTest
+    public void setUp() {
+        System.setProperty("webdriver.chrome.driver",
+                "C:\\Users\\Iuliia_Gorshkova\\Downloads\\chromedriver_win32\\chromedriver.exe");
+        webDriver = new ChromeDriver();
+        webDriver.manage().window().maximize();
+    }
+
+    @AfterTest
+    public void tearDown() {
+        if (webDriver != null) {
+            webDriver.close();
+        }
+    }
+
+    @Test
+    public void exerciseOneTest() {
+
+        //  1. Open test site by URL
+        webDriver.navigate().to("https://jdi-testing.github.io/jdi-light/index.html");
+
+        //  2. Assert Browser title
+        webDriver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
```

Comment:
Think about implicit waitings, maybe it could be replaced with explicit ones. You can use explicit waiting at some points when your test changes state of the page, when some components have to be loaded, when we just have navigated to the page, also as example: when we want to turn on the row of checkboxes we could wait explicitly the first one.